src/schematools/events/export.py

```diff
@@ -0,0 +1,143 @@
+"""Exporter module.
+
+This module facilitates the export of events from GOB database tables.
+The main objective of this module is to create full events that can be used
+to test the event-importer that is also part of the `schematools.events` package.
+
+The `schematools` cli interface had an entry to use the code in this module.
+"""
+
+from __future__ import annotations
+
+from collections import defaultdict
+from typing import Any, Generator
+
+from geoalchemy2.shape import to_shape
+from json_encoder import json
+from sqlalchemy import Table
+from sqlalchemy.engine import Connection
+
+from schematools.events import metadata
+from schematools.events.factories import tables_factory
+from schematools.types import DatasetSchema, DatasetTableSchema
+from schematools.utils import to_snake_case
+
+
+def fetch_complex_fields_info(dataset_table: DatasetTableSchema) -> dict[str, dict]:
+    """Collect info about complex fields (mainly relations)."""
+    complex_fields = {}
+
+    for field in dataset_table.fields:
+        multi = None
+        properties = {}
+        relation_ds = None
+        try:
+            if (nm_relation := field.nm_relation) is not None:
+                multi = True
+                properties = field["items"]["properties"]
+                relation_ds = nm_relation.split(":")[0]
+            if (relation := field.relation) is not None:
+                multi = False
+                properties = field["properties"]
+                relation_ds = relation.split(":")[0]
+        except KeyError:
+            continue
```

Comment:
Comment erbij? Waarom gebeurt dit, wat vang je hier af? Dit is haast "Errors should never pass silently".